From 6d7437e60526c8971c13db6663084a11515c1011 Mon Sep 17 00:00:00 2001
From: Marc-Antoine Perennou <Marc-Antoine@Perennou.com>
Date: Mon, 11 Jul 2011 14:40:50 +0200
Subject: [PATCH] build-sys: fix spaces versus tabs conflict

For some reason, tabs were replaced by spaces in commit
73ae0d5be6e070afe83d1a2324ac79941629201e

Signed-off-by: Marc-Antoine Perennou <Marc-Antoine@Perennou.com>
---
 term-utils/Makefile.am |    8 ++++----
 1 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/term-utils/Makefile.am b/term-utils/Makefile.am
index 5013898..7b545c3 100644
--- a/term-utils/Makefile.am
+++ b/term-utils/Makefile.am
@@ -68,12 +68,12 @@ if USE_TTY_GROUP
 if MAKEINSTALL_DO_CHOWN
 install-exec-hook::
 if BUILD_WALL
-       chgrp tty $(DESTDIR)$(usrbin_execdir)/wall
-       chmod g+s $(DESTDIR)$(usrbin_execdir)/wall
+	chgrp tty $(DESTDIR)$(usrbin_execdir)/wall
+	chmod g+s $(DESTDIR)$(usrbin_execdir)/wall
 endif
 if BUILD_WRITE
-       chgrp tty $(DESTDIR)$(usrbin_execdir)/write
-       chmod g+s $(DESTDIR)$(usrbin_execdir)/write
+	chgrp tty $(DESTDIR)$(usrbin_execdir)/write
+	chmod g+s $(DESTDIR)$(usrbin_execdir)/write
 endif
 endif
 endif
-- 
1.7.6.134.gcf13f6.dirty

